utils/storage.tsx

```diff
@@ -5,6 +5,7 @@ export const StorageKeys = {
   name: "name",
   email: "email",
   address: "address",
+  isRegistered: "registered",
```

Comment:
why not use enum?

ref: https://www.typescriptlang.org/docs/handbook/enums.html#string-enums

Reply:
Okay